compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/input/TextInputTests.kt

```diff
@@ -313,7 +328,7 @@ class TextInputTests : OnCanvasTests {
     }
 
     fun repeatedAccentMenuClicked() = runApplicationTest {
```

Comment:
If this test is intentionnaly disabled right now, then it's better to add `@Ignore` annotation rather than removing `@Test` annotation, so the intention to not run it is more obvious.